bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerFactoryImpl.java

```diff
@@ -54,12 +56,17 @@ public ActionManagerFactoryImpl() throws NotCompliantMBeanException {
         super(ActionManagerMBean.class);
         tasks = Collections.synchronizedMap(new LinkedHashMap<>());
     }
-    
+
     @Override
     public ActionManager createTaskManager(String name, ResourceResolver resourceResolver, int saveInterval) throws LoginException {
+        return this.createTaskManager(name, resourceResolver, saveInterval, ActionManagerConstants.DEFAULT_ACTION_PRIORITY);
+    }
+
+    @Override
+    public ActionManager createTaskManager(String name, ResourceResolver resourceResolver, int saveInterval, int priroty) throws LoginException {
         String fullName = String.format("%s (%s)", name, UUID.randomUUID().toString());
         
-        ActionManagerImpl manager = new ActionManagerImpl(fullName, taskRunner, resourceResolver, saveInterval);
+        ActionManagerImpl manager = new ActionManagerImpl(fullName, taskRunner, resourceResolver, saveInterval, priroty);
```

Comment:
Misspelled priority

Reply:
This is exactly the sort of high quality code review I expect.

Snark aside, will fix this, thanks for pointing it out.

Reply:
Hey, it's open-source.  You're getting so much more than you paid for already!  ;P  But seriously, thanks for the awesome PR!